TestApps/Sasquatch/Sasquatch/ViewControllers/MSCrashesViewController.swift

```diff
@@ -76,7 +76,7 @@ class MSCrashesViewController: UITableViewController, UIImagePickerControllerDel
       } else if (indexPath.row == 1) {
         cell.textLabel?.text = "Text Attachment";
         let text = UserDefaults.standard.string(forKey: "textAttachment")
-        cell.detailTextLabel?.text = (text?.characters.count ?? 0) > 0 ? text : "Empty";
+        cell.detailTextLabel?.text = (text?.count ?? 0) > 0 ? text : "Empty";
```

Comment:
This is the diff between Swift versions. And as far as I know, we are still using XCode 8. So, that is why CI build was failed.

Reply:
right...reverting this now.